test_2.4.sh

```diff
@@ -0,0 +1,3 @@
+#!/bin/bash
+
+datacats create --ckan=2.4 site3
```

Comment:
Do you think it's worth running any other tests across 2.3, 2.4 and master? How about the tweak commands since they "reach inside" and do things to the environment that might be broken by different ckan versions?